cockroachcloud/learn-about-pricing.md

```diff
@@ -15,8 +15,8 @@ RU and storage consumption is prorated at the following prices:
 
   Unit                    | Cost
   ------------------------|------
-  10M Request Units       | $1.00
-  1 GiB storage per month | $1.00
+  1M Request Units        | $1.00
```

Comment:
This isn't right - we're charging $1 per 10M RU's still, until June 1st.

Reply:
thanks for catching that, didn't mean to change it